frontend/templates/product-list/sections/FilterableProductsSection/facets/drawer/MenuFacetPanel.tsx

```diff
@@ -0,0 +1,57 @@
+import {
+   PRODUCT_LIST_DEFAULT_FACET_VALUES_COUNT,
+   PRODUCT_LIST_MAX_FACET_VALUES_COUNT,
+} from '@config/constants';
+import { ProductList, ProductListType } from '@models/product-list';
+import { useCallback } from 'react';
+import { MenuFacet } from '../MenuFacet';
+import { useCreateItemTypeURL } from '../useCreateItemTypeURL';
+import { useMenuFacet } from '../useMenuFacet';
+import { Panel } from './Panel';
+
+export type MenuFacetPanelProps = {
+   attribute: string;
+   productList: ProductList;
+   isOpen: boolean;
+   onClose: () => void;
+};
+
+export function MenuFacetPanel({
+   attribute,
+   productList,
+   isOpen,
+   onClose,
+}: MenuFacetPanelProps) {
+   const isDevicePartsItemType =
+      attribute === 'facet_tags.Item Type' &&
+      productList.type === ProductListType.DeviceParts;
+   const limit = isDevicePartsItemType
+      ? PRODUCT_LIST_MAX_FACET_VALUES_COUNT
+      : PRODUCT_LIST_DEFAULT_FACET_VALUES_COUNT;
+   const canLoadMore = limit < PRODUCT_LIST_MAX_FACET_VALUES_COUNT;
+
+   const menuFacet = useMenuFacet({ attribute, productList });
+   const createItemURL = useCreateItemTypeURL();
+   const handleItemClick = useCallback(
+      (value: string) => {
+         menuFacet.refine(value);
+         onClose?.();
+      },
+      [menuFacet, onClose]
+   );
+   return (
+      <Panel isOpen={isOpen}>
+         <MenuFacet
+            items={menuFacet.items}
+            limit={PRODUCT_LIST_DEFAULT_FACET_VALUES_COUNT}
```

Comment:
You're right, this can be confusing, I should probably rename `limit` to something different to avoid the confusion. This is not the same `limit` that's used by `useMenuFacet`. the `useMenuFacet` limit basically tells Algolia how many items to initially fetch, and for the item type Facet we are setting it to `200` because we want to fetch them all for SEO reasons. We still just show the user the first **X** items, and **X** is represented by the MenuFacet `limit` prop